### DISCUSSION TITLE
microsoft azure student

### DISCUSSION THREAD

**Discussion by Hiroohamada** [community/community#145771] [GitHub Education]
### Hi everyone,

![Screenshot (561)](https://github.com/user-attachments/assets/965f044b-b99e-4802-ae02-dec5c5531e7d)

Please help me solve this. I am getting error in availing my microsoft azure student pack.

**Comment by davevad93** [comment]
Hi @Hiroohamada ,

Several users encountered the same issue, but there's a possible solution:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).

I personally recommend the first option.
